package.json

```diff
@@ -11,7 +11,7 @@
     "lint:test": "jshint test/*.js",
     "prepublish:cdn": "npm run browserify",
     "publish:cdn": "gulp publish",
-    "test:unit": "istanbul cover _mocha -- --recursive",
+    "test:unit": "istanbul cover node_modules/mocha/bin/_mocha -- --recursive",
```

Comment:
Unfortunate that Windows requires this... but seems to still work elsewhere so seems fine.